### PR TITLE
Code syntax highlighting for sistent components

### LINKED ISSUE #5534: [Sistent] Code syntax highlighting for sistent components
### Current Behavior

Currently, the code shown for [sistent components](https://layer5.io/projects/sistent/components/button/code), is rendered as plain-text. 

### Desired Behavior
The JSX syntax should be highlighted.

### Screenshots / Mockups

Existing UI:
Screenshot 2024-05-12 at 14 22 07

### Implementation

we can utilize existing libraries for it.

### Acceptance Tests

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
-  🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by mannika763**
**Description**

This PR fixes #5534

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.
 

Result
image
image

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 3ec611d8291015fb519b61c7e9c40b50e50bc298 at: https://6694ca889464c837cab63f02--layer5.netlify.app

**Comment by sudhanshutech** [comment]
@mannika763 
Let's discuss this on Websites' call. Add this as an agenda item into the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit), if you would :)

**Comment by l5io** [comment]
🚀 Preview for commit 4ce2691689126981cf595b75db0590bf574fac8d at: https://6694e8859c1981746e6b3d72--layer5.netlify.app

**Comment by sudhanshutech** [comment]
have you not joined slack @mannika763 .? Please join here the meeting: https://meet.layer5.io/websites

**Comment by mannika763** [comment]
@sudhanshutech Sorry, I didn't see your message earlier. Can I join today?

**Comment by Ashparshp** [comment]
@mannika763 No sweat!! We don't have meeting for today, let's discuss in tomorrow development call.. add your item into meeting [minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?usp=drivesdk)

**Comment by l5io** [comment]
🚀 Preview for commit ac9142d358f349c06b6b85dd95a3804c24664888 at: https://669642ada7f8eb72265bad7f--layer5.netlify.app

File: src/components/CodeBlock/index.js [hunk 1]
```diff
@@ -54,7 +57,7 @@ const Code = ({ codeString, language }) => {
       {...defaultProps}
       code={codeString}
       language={language}
-      theme={theme}
+      theme={isDark ? theme : vsCodeLight}
```
Comment (sudhanshutech) [inline src/components/CodeBlock/index.js:60]: keep the theme as it was earlier please. Currently light doesn't seems to good so we can keep the existing effect

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Review by mannika763 — COMMENTED** [review]
I have added a condition for dark and light themes in the code-block.js file. This allows us to maintain the same theme for both light and dark modes according to the CodeBlock.js file.
Please let me know if this approach is correct or should i follow something else.
Thankyou

**Comment by l5io** [comment]
🚀 Preview for commit ae97610103396c994676c2027bd11c463f53ebbe at: https://669695f1f619af48744dcaeb--layer5.netlify.app

**Comment by sudhanshutech** [comment]
> I have added a condition for dark and light themes in the code-block.js file. This allows us to maintain the same theme for both light and dark modes according to the CodeBlock.js file. Please let me know if this approach is correct or should i follow something else. Thankyou

@mannika763 keep the same dark theme code block at light theme, we don't need separate logic or different theme for light it can be constant as it is dark right now.

**Review by mannika763 — COMMENTED** [review]
I have removed the conditional rendering related to isDark. Now, the dark theme will appear in both light and dark modes.

**Comment by l5io** [comment]
🚀 Preview for commit 612ca9673c3418cbdcfdfbf8b7289b536ed6828e at: https://669aadd4b2a5948e5e9e91b3--layer5.netlify.app

**Review by sudhanshutech — APPROVED** [review]

**Pull request merged**